src/app-layout/visual-refresh-toolbar/drawer/local-drawer.tsx

```diff
@@ -19,9 +19,13 @@ import styles from './styles.css.js';
 
 interface AppLayoutDrawerImplementationProps {
   appLayoutInternals: AppLayoutInternals;
+  widgetizedState: AppLayoutWidgetizedState;
```

Comment:
For backward compatibility, you need to use `Partial<AppLayoutWidgetizedState> | undefined` type here